lib/scheme/bitbucket.js

```diff
@@ -8,8 +8,8 @@ module.exports = {
   'issues/new?title': '/issues/new?title={title}',
   'wiki': '/wiki',
   'network': '/network',
-  'tree': '/src/{hash}{path}?at={hash}',
-  'blob': '/src/{hash}{path}?at={hast}',
+  'tree': '/branch/{hash}{path}?at={hash}',
+  'blob': '/branch/{hash}{path}?at={hast}',
```

Comment:
1. I found bitbucket url is include `src`, not `branch`.
2. `gitopen -b branch is because replace `{hash}` once, I will fix it by replace all.

Reply:
You may want `gitopen branches` command.